### PR TITLE
Creat work section and about myself

### PR DESCRIPTION
**Pull request by MayPyone**
Grid Layout and Flexbox are added to be more flexible. Button hovers are included and ul, li tags are also utilized effectively.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,41 +4,61 @@
...
+                    <li><a href="#"><img src="images/M.png"></a></li>
+                 </ul>
+
+                <div class="resume"><button type="button">Get my resume</button></div>
+
+                
+                            
+                </main>
                   
-    </header>
```
Comment (hiromataba) [inline index.html:16]: -  [ ] For the about page you should use a `section` tag  as required to wrap your about section

File: index.html [hunk 2]
```diff
@@ -4,41 +4,61 @@
...
+                        <li><img src="images/html.png"/><div class="language_name">HTML</div></li>
+                        <li><img src="images/css.png"/><div class="language_name">CSS</div></li>
+                    </ul>
+                </div>
+                <div class="stylist">
+                    <p class="framework">Frameworks <img class="dropdown" src="images/Icon - Chevron - Right - Larger.png"/> </p>
+                    <div class="separator"><img src="images/Separator Bottom.png"/></div>
+                     <p class="skills">Skills <img class="dropdown2" src="images/Icon - Chevron - Right - Larger.png"/></p>
+                     <div class="separator"><img src="images/Separator Bottom.png"/></div>
+                </div>
```
Comment (hiromataba) [inline index.html:57]: - [ ] For The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li as required

File: index.html [hunk 3]
```diff
@@ -4,41 +4,61 @@
...
-            <nav class="menu_icon"><img src="images/Union.png"/></nav>
+        <section>
+            <div class="headline">
+                <main>
+                    <h1>
+                        About Myself
+                    </h1>
+                    
+                    <p>
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
```
Comment (hiromataba) [inline index.html:20]: - [ ] Only your about section is displayed now , please make the home page and the other section like work section visible ,  this milestone is to finish the  mobile version

- Your design 👇 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @MayPyone    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -5,40 +5,265 @@
...
+                  
+            </div>
+            
+        </section>
+
+
+
+
+
+    <div class="grid_container">
```
Comment (tufoinnkuo10) [inline index.html:47]: - [optional] please kindly put the works section in a section tag as well and not a div like i explained for the about section. nice work.

File: index.html [hunk 5]
```diff
@@ -5,40 +5,265 @@
...
+
+        
+        <div class="project_tag"><button>See project</button></div>
+       
+    </div>
+</div>
+
+</div>
+    </div>
+    <div class="aboutme">
```
Comment (tufoinnkuo10) [inline index.html:206]: - [ ] please like the previous reviewer noted, The container of the section is a section tag. a section is a semantic element for creating standalone sections in a web page. this will be of great use. kindly remove the header tag from the about section as it represents introductory content, typically a group of introductory or navigational aids.
eg 
```
<section>
      .. about section
</section>
```

Comment (MayPyone) [reply index.html:206]: Dear sir, thanks for your fruitful comments. So should I use a section tag instead of a div name called 'aboutme'?

File: index.html [hunk 6]
```diff
@@ -5,40 +5,263 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css"/>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
```
Comment (shahadat3669) [inline index.html:10]: - [ ] [Optional] Great job on your work! However, there is a small issue with the vertical scroll bar appearing on your webpage. According to the design, there should be no vertical scroll bar visible. Please review and adjust the width and spacing of your elements to ensure that the content fits within the available space without triggering the appearance of a vertical scroll bar. By resolving this issue, you will achieve a closer match to the intended design and improve the overall user experience. 👍 

[Document (9).webm](https://github.com/MayPyone/My_Portfolio/assets/55840999/749828a6-39ac-4b2d-bbde-5a3b6b8a3c8e)

File: index.html [hunk 7]
```diff
@@ -5,40 +5,263 @@
...
+                  </div>
+          
+
+                
+                <div class="project_tag"><button>See project</button></div>
+               
+            </div>
+        </div>
+        
+ </div>
```
Comment (shahadat3669) [inline index.html:86]: - [ ] [Optional] Great! Your card looks good. However, I would suggest using the `<article>` tag instead of a `<div>` tag for the main card wrapper. The `<article>` tag is more semantically appropriate in this context. You can refer to [this documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to learn more about its usage. Please consider updating your code accordingly.

File: index.html [hunk 8]
```diff
@@ -5,40 +5,263 @@
...
+        <strong class="stronger"> Multi-Post
+        <br>
+        Stories
+    </strong>
+      </h2>
+    
+        <div class="canopy">
+            <ul>
+                <li><div class="canopy1">CANOPY</div></li>
+
```
Comment (shahadat3669) [inline index.html:101]: - [ ] [Optional] Upon reviewing your code, I noticed that there are several instances of unnecessary whitespace. I would recommend removing the extra spaces to make your code cleaner and more readable. It is important to maintain consistent formatting throughout your codebase. Please take some time to remove the extra spaces. 👍

File: index.html [hunk 9]
```diff
@@ -5,40 +5,263 @@
...
+                    <h2><strong class="strong">About Myself</strong>
                         
-            </main>
-              
+                    </h2>
+                    
+                    <p>
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <div class="connect">LET'S CONNECT</div>
```
Comment (shahadat3669) [inline index.html:218]: - [ ] [Optional] Well, your social icon titles look visually good. However, instead of using a `<div>` tag, it would be more semantically appropriate to use a heading tag, such as `<h2>` or `<h3>`, to represent the title. This will improve the structure and accessibility of your code. Please consider updating it accordingly. Thank you!

**Review by shahadat3669 — APPROVED** [review]
Hi @MayPyone, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Descriptive pull request message
✅ linter error checks validated

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### You can consider:
- [ ]  [Optional] Your base branch name is currently set as "master". According to the recommended practice of [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md), it is advised to use the branch name "main" as the default base branch. In your future projects, please consider using "main" as the base branch to align with the best practices. Thank you for your attention to this matter.
- [ ] [Optional] Upon reviewing your project, I noticed that you mentioned a technology that is not related to your current project. This can be confusing for visitors and may lead to a misinterpretation of your project's purpose. Please consider updating the information to accurately reflect the technologies used in your project. 👍
image
- [ ] [Optional] Great! I see that you have a license section in your readme file, which is a good practice. However, the link you provided for the MIT license redirects to a generic MIT license template. To ensure clarity and compliance, I recommend adding a license file specific to your project and personalizing it accordingly. This will provide clear licensing information for your project and allow others to understand the terms and conditions for its use. Keep up the good work! 👍
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
